lib/classes/CLI.js

```diff
@@ -330,8 +330,20 @@ class CLI {
 
   getVersionNumber() {
     this.consoleLog(
-      `Framework Core: ${version}\nPlugin: ${enterpriseVersion}\nSDK: ${sdkVersion}\n`
+      `\nFramework Core: ${version}\nPlugin: ${enterpriseVersion}\nSDK: ${sdkVersion}`
     );
+
+    const userNodeVersion = Number(process.version.split('.')[0].slice(1));
+    // only show components version if user is running Node 8+
+    if (userNodeVersion >= 8) {
+      const { cliVersion, coreVersion } = require('@serverless/cli');
+      this.consoleLog(
+        `Components Core: ${coreVersion}\nComponents CLI: ${cliVersion}\n`
+      );
+    } else {
+      this.consoleLog('');
```

Comment:
I find this end line padding controversial as well,  still we have that one in master already

Reply:
👌